pandas/tests/indexes/datetimes/test_ops.py

```diff
@@ -307,6 +307,28 @@ def test_drop_duplicates_metadata(self):
         tm.assert_index_equal(idx, result)
         assert result.freq is None
 
+    def test_drop_dst_boundary(self):
```

Comment:
can you move near the other test in test_timezones.py ?

Reply:
Sure thing.  Done.